### ISSUE TITLE
Not compatible with th-abstraction-0.5

### ISSUE THREAD

**Issue by ysangkok** [tomjaguarpaw/product-profunctors#68]
product-profunctors is one of the very last packages blocking

* https://github.com/commercialhaskell/stackage/issues/6897

So it would be awesome if it were updated.

**Comment by tomjaguarpaw** [comment]
Hmm, strange. I only got an email notification about this 12 hours ago.

**Comment by ysangkok** [comment]
Are you talking about the GitHub notification? Possibly they have some kinda rate limiting going on. Repositories like Nixpkgs and Stackage are kind of a weird fit for GitHub.

**Comment by tomjaguarpaw** [comment]
Yes, but I can see what happened: it wasn't until I published `product-profunctors-0.11.1.0` that the upper bound on `th-abstraction` was even present, so I wasn't @-mentioned in the original message.

**Comment by tomjaguarpaw** [comment]
Please close this issue if you believe everything has been dealt with.
